smf/external-dns/config.toml

```diff
@@ -11,5 +11,5 @@ path = "/dev/stdout"
 if_exists = "append"
 
 [storage]
-storage_path = "/var/oxide/dns"
+storage_path = "/data"
```

Comment:
`/data/dns` (in case we ever want to put anything else there)?

Reply:
Done in 02e52b4741c5a499690b29c77b77e7614a9536a8